pkg/controller.v1alpha3/experiment/experiment_controller.go

```diff
@@ -18,6 +18,7 @@ package experiment
 
```

Comment:
Do we want to make this change v1alpha3 version since we already created a release?
What do you think @gaocegege @johnugeorge ?

Reply:
Depends on if we maintain v1alpha3 in the next release. I think we should keep for 1 more release since the v1beta1 may be not stable. WDYT @johnugeorge

Reply:
cc @idahoakl

PTAL, thanks.

Reply:
@idahoakl PTAL, thanks.